pkg/console/operator/operator.go

```diff
@@ -140,15 +140,7 @@ func NewConsoleOperator(
 
 // key is actually the pivot point for the operator, which is our Console custom resource
 func (c *consoleOperator) Key() (metav1.Object, error) {
-	operatorConfig, err := c.operatorConfigClient.Get(api.ConfigResourceName, metav1.GetOptions{})
-	if errors.IsNotFound(err) && CreateDefaultConsoleFlag {
```

Comment:
Can we remove the create default console flag?

Reply:
We are still creating out top level `console` config.
Just not the `operator` config.

Reply:
But thats a good Q.  Should both be created by CVO now instead of the operator? If so, we could eliminate the flag.

Reply:
Looks like it's already there?

https://github.com/openshift/cluster-config-operator/blob/master/empty-resources/0000_05_config-operator_02_console.cr.yaml

Reply:
`/empty-resources` is new to me.  Great. I think we can delete the other `func` as well, and the flag.

Reply:
Both removed.  We now trust someone else will recreate all of our configs.